src/zope/index/field/index.py

```diff
@@ -84,8 +76,9 @@ def index_doc(self, docid, value):
     def unindex_doc(self, docid):
         """See interface IInjection"""
         rev_index = self._rev_index
-        value = rev_index.get(docid)
```

Comment:
Maybe use a marker value, to avoid doing both a `__contains__` and then a `__getitem__` call?

```python
value = rev_index.get(docid, _MARKER)
if value is _MARKER:
   return # not in index
...
```